src/pages/2018.js

```diff
@@ -1,157 +1,224 @@
 import React from 'react';
+import Layout from '../components/indexLayout'
 import Report from '../components/reportLayout';
+import Img from 'gatsby-image'
+import { graphql } from 'gatsby'
 
-const state18 = () => (
-  <Report>
-    <div className="page">
-      <section className="2018-Report">
-        <div className="container">
-          <h1>2018 Yearly Report</h1>
-            <p>2018 was a truly memorable year for us, at Sahyadri Open Source Community.<br />
-            We watched our community increase in size and many members developed and learnt new skills;<br />
-            We also provided mentorship, organised a handful of Events, Workshops, and an OverNight Hackathon!<br /><br />
-            Here's a report of SOSC's 2018:
-              <ul>
-                <li><h1>Rustathon 2k18</h1>
-                  <br />
-                  <br />10th February 
-                  <br />
-                  <br />Rustathon 2k18 was the first workshop of the year for SOSC, based on the blazing fast RUST programming language
-                        organized by the Mozilla community. This event was a part of the MozActivate campaign of Mozilla. 
-                        The speaker of this event was Mr Jayesh Katta Ramalingaiah, A Mozilla Representative and Full Stack Developer;
-                        upto 10,000$ worth of Amazon Web Service(AWS) credits were sponsored by Amazon Solely for Rust-AWS Projects.
-                </li>
-                <br />
-                <li><h1>DevHost:18</h1>
-                 <br />
-                  <br />04th and 5th May 
+const state18 = (props) => (
+  <Layout>
+    <Report>
+      <div className="page">
+        <section className="2018-Report" style={{
+          fontSize: '20px',
+        }}>
+          <div className="container">
+          <Img
+              fluid={props.data.imageFirst.childImageSharp.fluid}
+              className="sosc-logo"
+              alt="sosc logo"
+              style={{
+                height:'10%',
+                width:'10%',

```

Comment:
This is not an efficient way to supply the image size, since the % is calculated based on the device height/width.

Reply:
gotcha, what would be a better alternative?